cmd/yawol-cloud-controller/main.go

```diff
@@ -352,3 +365,12 @@ func getInfrastructureDefaultsFromEnvOrDie() targetcontroller.InfrastructureDefa
 		InternalLB:       pointer.Bool(internalLb),
 	}
 }
+
+func (i *loadbalancerClassNames) String() string {
+	return strings.Join(*i, ",")
+}
+
+func (i *loadbalancerClassNames) Set(value string) error {
+	*i = append(*i, value)
```

Comment:
IIUC, you need to add multiple `--classname` flags if you want to specify multiple classes.
Does `--classname=a,b` work as well?

Reply:
Yes, if you need multiple classes than it is needed to add this flag multiple times.
Not in this case. If we need this than we also need to parse.

Reply:
Ok, then let's keep it this way. IIRC, the https://github.com/spf13/pflag package can do this out-of-the-box